airflow/providers/amazon/aws/log/cloudwatch_task_handler.py

```diff
@@ -33,6 +33,24 @@
     from airflow.models import TaskInstance
 
 
+def json_serialize_legacy(o):
+    """Json serializer replicating legacy watchtower behavior.
+
+    The legacy `watchtower@2.0.1` json serializer function that serialized
+    datetime objects as ISO format andall other non-JSON-serializable to `null`.
+    """
+    if isinstance(o, (date, datetime)):
+        return o.isoformat()
+
+
+"""json serializer replicating current watchtower behavior
```

Comment:
```suggestion
"""
json serializer replicating current watchtower behavior
```

I'm surprised the style checks didn't get angry at this, perhaps that is one of the checks that are not yet enabled, but first line of the docstring needs to be on a newline.

Reply:
fixed